### PR TITLE
i18n(de-DE): Add markdown mode to config reference

### PR DESCRIPTION
**Pull request by hippotastic**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Adds even more recent changes to the German version of our configuration reference page. Those were added between filing and merging of my previous PR! 😄

### CODE REVIEW

File: src/pages/de/reference/configuration-reference.md [hunk 1]
```diff
@@ -262,6 +262,28 @@ Eine Markdown-Seite wird als Entwurf betrachtet, wenn sie die Frontmatter-Eigens
 ```
 
 
+### markdown.mode
+
+<p>
+
+**Typ:** `'md' | 'mdx'`<br>
+**Standard:** `mdx`
+</p>
```
Comment (delucis) [inline src/pages/de/reference/configuration-reference.md:271]: Do we still need the `<p>` wrapper for this? I think this should work too:

```suggestion
**Typ:** `'md' | 'mdx'`<br>
**Standard:** `mdx`
```

**Review by delucis — COMMENTED** [review]
One question about the formatting, but content LGTM!

File: src/pages/de/reference/configuration-reference.md (hunk 1 above)
Comment (delucis) [reply src/pages/de/reference/configuration-reference.md:271]: OK!

Comment (hippotastic) [reply src/pages/de/reference/configuration-reference.md:271]: If it helps: I can only *barely* tolerate it because it's NWTWWHB!

**Pull request merged**